SA3/qa_web/search_indexes.py

```diff
@@ -0,0 +1,21 @@
+__author__ = 'jet'
+# import datetime  # RuntimeWarning: naive datetime is not the best time marker in django
+from django.utils import timezone
+from haystack import indexes
+from qa_web.models import Questions
+
+# If it's the first time you try to use search, please execute: python manage.py rebuild_index
```

Comment:
This information should be found under the README's Quickstart section, not here.